### PR TITLE
External user create request

### LINKED ISSUE #946: External user create request
1. Create a new page for these users
2. Add a checkbox to organisation 'external' 
3. Add organisation to user requests
4. Add a model for corporate domains
5. Show request author in user requests

### PR DESCRIPTION
**Pull request by tsumo**
## PR includes

-   [x] Feature

## Related issues

Resolve #946

### CODE REVIEW

File: src/modules/userCreationRequestSchemas.ts [hunk 1]
```diff
@@ -1,16 +1,66 @@
...
+    surname: z.string().min(1, { message: tr('Minimum {min} symbols', { min: 1 }) }),
+    firstName: z.string().min(1, { message: tr('Minimum {min} symbols', { min: 1 }) }),
+    middleName: z.string().optional(),
+    email: z.string().min(5, { message: tr('Minimum {min} symbols', { min: 5 }) }),
+    phone: z.string().min(5, { message: tr('Minimum {min} symbols', { min: 5 }) }),
+    login: z.string().min(1, { message: tr('Minimum {min} symbols', { min: 1 }) }),
+    accountingId: z.string().optional(),
+    organizationUnitId: z.string(),
+    groupId: z.string().nullish(),
+    supervisorId: z.string().nullish(),
```
Comment (nyet-ty) [inline src/modules/userCreationRequestSchemas.ts:16]: we have supervisorLogin String in schema in model UserCreationRequest

**Review by nyet-ty — CHANGES_REQUESTED** [review]

File: src/modules/userCreationRequestSchemas.ts (hunk 1 above)
Comment (nyet-ty) [reply src/modules/userCreationRequestSchemas.ts:16]: oh, my bad, it's ok

File: src/components/CreateUserModal/CreateUserModal.i18n/ru.json [hunk 2]
```diff
@@ -1,22 +1,6 @@
...
-    "Login": "Логин",
-    "Organization:": "Организация",
-    "Create external account:": "Создать внешнюю учетную запись",
-    "Accounting id": "Id в бухгалтерской системе",
-    "Date": "Дата",
-    "Role": "Роль",
-    "OS preference": "Предпочтения ОС",
-    "Create corporate email:": "Создать корпоративный email",
-    "Corporate email": "Корпоративный email"
+    "Create request for user creation": "",
```
Comment (nyet-ty) [inline src/components/CreateUserModal/CreateUserModal.i18n/ru.json:2]: You forgot translations here

Comment (tsumo) [reply src/components/CreateUserModal/CreateUserModal.i18n/ru.json]: fixed

**Review by nyet-ty — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**